### ISSUE TITLE
Redirect Request: R&S FAQ for WAVE

### ISSUE THREAD

**Issue by jennymayoco** [department-of-veterans-affairs/va.gov-team#87445]
### Status
**[2024-07-18] [Fran]** Asked Chris [in Slack](https://agilesix.slack.com/archives/C05Q7J61NR5/p1721327760834579) to async refine so it can go into Sprint 9.
**[2024-07-15] [Fran]** Targeting Sprint 9 (Jul 24 - Aug 6)
**[2024-07-15] [Jenny]** R. Hecht has confirmed that the static pages with the new tool are published so these redirects are good to go! We'll archive the R&S page shortly after or close to the redirect being implemented so folks don't get a 404 page in the meantime.  We won't be doing a content audit on this redirect because all references to WAVE have been updated.
**[2024-07-02] [Jenny]** We are archiving the R&S FAQ page on WAVE as this tool will be replaced by the new Verify your Enrollment tool. The team's latest update is that they intend to launch to prod today. Once we get confirmation, Randi H. will be publishing the static page with the widget and we'll be able to move forward with this redirect. We just wanted to kick off this process since the tool should be launching soon. I'll follow up once we're good to move forward with implementation. 

### Instructions
- Requests must be submitted **AT LEAST** 2 weeks in advance.  Some requests take a significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
- This issue will be used from initial request through implementation to ensure all individuals working on this are notified of status updates.  Please do not create multiple issues to track differnt steps.
- It is your responsibility to notify VA stakeholders as appropriate.  

| POC | Github alias(es) |
--- | --- 
| Product team point(s) of contact: | @cassidy-beach @jason-gcio |
| Content point of contact: | @RLHecht |
| IA point of contact: | @jennymayoco  |

### Type of request
- [x] We are retiring or taking down a page and need to redirect the URL (complete redirect section)
- [ ] We are changing the URL of an existing page (complete redirect section)
- [ ] We need a custom vanity URL (complete vanity URL section)
- [ ] We are removing a temporary redirect  (add relevant background and link to original redirect request)

### Do you need Public Websites to implement this redirect?
- [x] Yes
- [ ] No - Please note: Redirect still needs approval from IA and PW will do PR review

### Implementation date
When does this request need to be live? **NOTE:** Redirects deploy M-Th. Friday launches should be avoided.
Choose one:
- [X] On a specific date, connected to a program or publishing requirement. **Date TBD** From team 7/1: "I believe we intend to be in prod tomorrow under a feature flag"
- [ ] Any day prior to a deadline/latest possible date. **DEADLINE:** 
- [ ] Any day after an earliest possible date. **EARLIEST DATE:**
- [ ] During a specific date range: ***EARLIEST DATE TO LATEST DATE:***
- [ ] No hard timing requirements, publish when possible.

### Redirects 
Current URL  |  Redirect Destination or New URL
---  |  ---
https://www.va.gov/resources/gi-bill-wave-faqs/ | ~~http://www.va.gov/education/verify-school-enrollment/~~ https://www.va.gov/resources/gi-bill-enrollment-verification-faqs/

JM 7/24 - updated new URL to a new R&S page that better matches the content on the old URL

### Vanity URLs 
Vanity URL  |  Landing page
---  |  ---
requested URL | landing page URL

**Link to campaign landing page request issue:**

### Process, Roles and Responsibilities
- [ ] Requesting team: Above information is provided
- [ ] Requesting team: All appropriate VA stakeholders are notified as appropriate
- [x] IA: Request is vetted and documented and implementation plan is clear 
- [x] IA: Request is assigned to appropriate team for implementation 
- [ ] Implementation team: Work is complete
- [ ] Implementation team: Validated in production
- [ ] Requesting team: Validates everything is correct in production and closes ticket

**Comment by jennymayoco** [comment]
Hi @jilladams @FranECross! We are archiving the R&S FAQ page on WAVE as this tool will be replaced by the new Verify your Enrollment tool. The team's latest update is that they intend to launch to prod today. Once we get confirmation, Randi will be publishing the static page with the widget and we'll be able to move forward with this redirect. We just wanted to kick off this process since the tool should be launching soon. I'll follow up once we're good to move forward with implementation. Thanks!

cc @RLHecht @cassidy-beach @jason-gcio for awareness

**Comment by jennymayoco** [comment]
Hi @jilladams @FranECross, @RLHecht has confirmed that the static pages with the new tool are published so these redirects are good to go! We'll archive the R&S page shortly after or close to the redirect being implemented so folks don't get a 404 page in the meantime.

We won't be doing a content audit on this redirect because all references to WAVE have been updated.

**Comment by jennymayoco** [comment]
Hi @chriskim2311 @FranECross so sorry for the this last minute change, but there's a new R&S page that we feel would be a better match for the WAVE FAQ page. Are we able to update in this ticket and have the WAVE page redirect to `https://www.va.gov/resources/gi-bill-enrollment-verification-faqs/` instead or should I submit a new one?

cc @bethpotts @aprocik1

**Comment by chriskim2311** [comment]
Not a problem, I will try to get that merged today as the other PR has already merged. Thanks for letting us know!

**Comment by jennymayoco** [comment]
thank you so much! I've updated the ticket with the correct URL.

Apologies for the last minute change!

**Comment by jennymayoco** [comment]
Hi @chriskim2311! I'm seeing that this page: https://www.va.gov/resources/gi-bill-enrollment-verification-faqs/ is redirecting to http://www.va.gov/education/verify-school-enrollment/. My original ask was if we were able to update the new destination to the R&S page so the final redirect request here would be: 

Current URL  |  Redirect Destination or New URL
---  |  ---
https://www.va.gov/resources/gi-bill-wave-faqs/ | ~~http://www.va.gov/education/verify-school-enrollment/~~ https://www.va.gov/resources/gi-bill-enrollment-verification-faqs/

The new `/resources/gi-bill-enrollment-verification-faqs/` page matches the content on the `/resources/gi-bill-wave-faqs/` page and so we wanted to update the destination so it wasn't the `/verify-school-enrollment/` page anymore. Sorry for the confusion!

**Comment by jilladams** [comment]
@jennymayoco Chris is working on this fix, thanks for flagging. There's no auto deploy of redirects on Fridays, so this fix would naturally roll out on Monday. Is that ok, or should we plan to ask for an Out of band deploy for this fix?

**Comment by jennymayoco** [comment]
hi @jilladams thanks for the update! This can wait until the next deploy on Monday.

**Comment by jennymayoco** [comment]
hi @jilladams @chriskim2311! I wanted to check and see if there any updates on this redirect

**Comment by jilladams** [comment]
Hmm -- Chris merged the change to fix it, but it looks like today's daily deploy failed: https://github.com/department-of-veterans-affairs/vsp-platform-revproxy/actions/workflows/deploy-daily.yaml

Thanks for pinging - will follow up with Devops.

**Comment by jilladams** [comment]
@jennymayoco the deploy got through - the page is redirecting correctly for me now. Feel free to close if / when you're able to verify.

**Comment by jennymayoco** [comment]
@jilladams I'm still seeing that both links `https://www.va.gov/resources/gi-bill-wave-faqs/` and `https://www.va.gov/resources/gi-bill-enrollment-verification-faqs/` are redirecting to VYE page.

**Comment by chriskim2311** [comment]
Hi @jennymayoco `https://www.va.gov/resources/gi-bill-wave-faqs/ ` is redirecting to `https://www.va.gov/resources/gi-bill-enrollment-verification-faqs/` This is expected right?

**Comment by jennymayoco** [comment]
hi @chriskim2311! yes, that's the expected redirect

**Comment by chriskim2311** [comment]
@jennymayoco Thanks that redirect is working on my end. Also https://www.va.gov/resources/gi-bill-enrollment-verification-faqs/ is not redirecting anywhere. What issue are you seeing?

**Comment by jennymayoco** [comment]
oh! I'm not sure what's happening on my end 😅 this page https://www.va.gov/resources/gi-bill-enrollment-verification-faqs/ is redirecting to https://www.va.gov/education/verify-school-enrollment/, as is https://www.va.gov/resources/gi-bill-wave-faqs/

I just closed out of my browser though and when I checked the links again, they're going to the right destination. sorry for the confusion there! not sure why that kept happening. thank you both for your assistance and support with this!

Confirming redirects are correct so closing ticket
